app/controllers/api/base_controller.rb

```diff
@@ -9,12 +9,13 @@ class BaseController < ActionController::Base
     before_action :set_default_response_format, :authorize, :set_taxonomy
     before_action :check_media_type
     before_action :assign_lone_taxonomies, :only => :create
-    before_action :add_info_headers, :set_gettext_locale
+    before_action :set_gettext_locale
     before_action :session_expiry, :update_activity_time
     around_action :set_timezone
 
     respond_to :json
 
+    after_action :add_info_headers
```

Comment:
Can you explain why this needs to change from a before action to an after action?